backends/nxp/backend/edge_helper.py

```diff
@@ -125,3 +125,21 @@ def previous_non_qdq_node(node: Node, input_index: int = 0) -> Node | None:
             current_node = current_node.args[0]
         else:
             return current_node
+
+
+def get_non_qdq_users(node: Node) -> list[Node]:
```

Comment:
I don't understand the logic here. You are looking the list of node uses of users (i.e. not direct users but list of users of the first users node)?
Why?